bitrise.yml

```diff
@@ -2299,4 +2299,4 @@ trigger_map:
 meta:
   bitrise.io:
     stack: osx-xcode-16.2.x
-    machine_type_id: g2.mac.large
+    machine_type_id: g2.mac.4large
```

Comment:
Changing only the default stack for now

Reply:
flaky unit tests are with xlarge not with large.
We can see this running and working while we see how to manage those unit tests failures with xlarge